dropkick/src/main/java/life/genny/dropkick/live/data/InternalConsumer.java

```diff
@@ -214,18 +214,14 @@ public void getEvent(String event) {
 		log.info("DROPDOWN :Loaded " + msg.getItems().size() + " baseentitys");
 
 		for (BaseEntity item : msg.getItems()) {
-			String name = null;
+			String name = item.getName();
```

Comment:
if item.getName() is null would it not make sense to check PRI_NAME? (Compatibility)
Further we could make a more descriptive log then logging the same thing but as a warning to suggest that the base entity name is null.